docker-compose.admin-panel.yml

```
@@ -2,12 +2,12 @@ version: "3.6"
 
 services:
   mqtt-server:
-    image: ghcr.io/shankari/everest-demo/mqtt-server:0.0.1
+    image: ghcr.io/everest/everest-demo/mqtt-server:0.0.1
```

Let's keep this in the `US-JOET` organization until the new image building process is in place

```suggestion
    image: ghcr.io/us-joet/everest-demo/mqtt-server:build_pkg_task_1_features
```

---

@shankari I was planning to updated the remaining in their respective branches.

---

@shankari can you revert the extraneous changes then? I don't want to have a lot of flip-flopping that is merged, and it clutters up the review.